libs/jsonforms-components/src/lib/Controls/FormStepper/FormStepperControl.tsx

```diff
@@ -126,6 +134,53 @@ export const FormStepper = ({
     );
   };
 
+  const resolveLabelFromScope = (scope: string) => {
+    const lastSegment = scope.split('/').pop()!;
```

Comment:
please do you not use .pop()!... This is telling typescript you know when you pop the string array there will be a value.  But this may be incorrect at runtime when the code is being executed.

Use a if statement to check if there is a value when popping the string array and put a default value for 'lastSegment' if it doesnt go into the if statement.

Reply:
Make sure when you write your unit tests that you take this "use case" into account.